Vuzix/app/src/main/java/com/facilitation/view/DynamicThemeReceiver.kt

```diff
@@ -0,0 +1,16 @@
+package com.facilitation.view
+
+import android.appwidget.AppWidgetManager
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+
+
+class DynamicThemeReceiver : BroadcastReceiver() {
+
+    override fun onReceive(context: Context, intent: Intent) {
```

Comment:
Do you actually need intent parameter here? It does not seem it is being used

Reply:
Not sure yet. It's from the template app